client/src/store/api.js

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Copyright (c) 2019-present for NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License ");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { Endpoint } from '../config/'
+import helper from '../helper'
+
+export default {
+    namespaced: true,
+    state: {
+    
+    },
+    getters: {
+        nodeList: state => Endpoint.nodes,
+        currentNode: state => Endpoint.api.replace('http://', '').replace(':3000', ''),
```

Comment:
will not work with `https://`, please fix

Reply:
Agree. I will fix it